AirCasting/SearchAndFollow/CompleteScreen/StaticSingleMeasurementView.swift

```diff
@@ -0,0 +1,56 @@
+// Created by Lunar on 28/02/2022.
+//
+
+import SwiftUI
+
+struct StaticSingleMeasurementView: View {
+    @Binding var selectedStreamId: Int?
+    var streamId: Int
+    var streamName: String
+    var value: Double
+    
+    var body: some View {
+        VStack(spacing: 3) {
+            Button(action: {
+                selectedStreamId = streamId
+            }, label: {
+                VStack(spacing: 1) {
+                    Text(showStreamName(streamName))
+                        .font(Fonts.systemFont1)
+                        .scaledToFill()
+                        HStack(spacing: 3) {
+                            dot
+                            Text("\(Int(value))")
+                                .font(Fonts.regularHeading3)
+                                .scaledToFill()
+                        }
+                        .padding(.vertical, 4)
+                        .padding(.horizontal, 9)
+                        .overlay(
+                            RoundedRectangle(cornerRadius: 8)
+                                .strokeBorder((selectedStreamId == streamId) ? Color.aircastingGray : .clear)
+                        )
+                    }
+            })
+        }
+    }
+    
+    var dot: some View {
+        Color.aircastingGray
+            .clipShape(Circle())
+            .frame(width: 5, height: 5)
+    }
+    
+    func showStreamName(_ streamName: String) -> String {
+        streamName
+            .replacingOccurrences(of: ":", with: "-")
+            .drop { $0 != "-" }
+            .replacingOccurrences(of: "-", with: "")
```

Comment:
this also looks like it wants to be in a viewmodel. Also - don't we use this exact code all over the app? Maybe we should make an extension for it or smth?